pkgs/development/tools/changie/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "changie";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "miniscruff";
+    repo = pname;
+    sha256 = "sha256-VzrSfigpkOvgywq0dHIXZS2If8qc8HCo51FzopKORwM=";
+  };
+
+  vendorSha256 = "sha256-+Q0vNMd8wFz+9bOPfqdPpN2brnUmIf46/9rUYsCTUrQ=";
+
+  meta = with lib; {
+    homepage = "https://changie.dev";
+    description = "Automated changelog tool for preparing releases with lots of customization options";
+    license = licenses.mit;
+    platforms = platforms.unix;
+    maintainers = with maintainers; [ ];
```

Comment:
Maintainer is needed, see https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes

Reply:
Woops, didn't see that it was empty! Fixed, thanks for pointing out!